### ISSUE TITLE
Set up training environment for for initial engagement prediction model

### ISSUE THREAD

**Issue by raindrift** [greenearth-social/ingex#8]
I think this work is well underway. Let's discuss where it's at and see if we can split this into some more useful set of tasks.

Steps in this story:
- set up VM in Google Cloud 
- put Will's code into GreenEarth repo
- get it to running state on old training data

Steps for future stories:
- get ingest working
- training run
- evaluate engagement prediction accuracy, just likes so far
  -  iterate until we get something better than just politics and porn
- evaluate per-user engagement prediction vs. available user data (cold start, overfitting, etc.)
- update to multi-task model that also predicts replies, quotes

Useful links:
- facebook world2vec: https://x.com/jonathanstray/status/1820203461599985830?s=46&t=cjKMpIess6tvHgbjU0rbAw
- how X's recommender works: https://blog.x.com/engineering/en_us/topics/open-source/2023/twitter-recommendation-algorithm
- X's code: https://github.com/twitter/the-algorithm
